cypress/e2e/email-address.cy.js

```diff
@@ -0,0 +1,67 @@
+describe("email address", () => {
+  before(() => {
+    cy.visit("/email-address");
+  });
+
+  context("back link", () => {
+    beforeEach(() => {
+      cy.get('[data-testid="back-link"]').as("button");
+    });
+
+    it("displays", () => {
+      cy.get("@button").should("be.visible");
+    });
+
+    it("says 'Back'", () => {
+      cy.contains("Back");
+    });
+
+    it("links to the right page", () => {
+      cy.get("@button").click();
+      cy.url().should("include", "date-of-birth");
+    });
+  });
+
+  context("email field", () => {
+    beforeEach(() => {
+      cy.get('[data-testid="email-field"]').as("field");
```

Comment:
Here you are trying to use the same test ID as two different things. In the HTML, it doesn't seem like you've labeled anything as `data-testid=email-field-label`.

Reply:
I believe that I did on line 87 ?

Reply:
I've added `data-testid=email-field-label`